safe_transaction_service/history/services/balance_service.py

```diff
@@ -268,50 +263,28 @@ def get_usd_balances(
         exclude_spam: bool = False,
     ) -> List[BalanceWithFiat]:
         """
-        All this could be more optimal (e.g. batching requests), but as everything is cached
-        I think we should be alright
+        NOTE: PriceService was removed, this function return balances with price 0.
 
         :param safe_address:
         :param only_trusted: If True, return balance only for trusted tokens
         :param exclude_spam: If True, exclude spam tokens
         :return: List of BalanceWithFiat
         """
-        # TODO Use price service get_token_cached_usd_values
         balances: List[Balance] = self.get_balances(
             safe_address, only_trusted, exclude_spam
         )
-        try:
-            eth_price = self.price_service.get_native_coin_usd_price()
-        except CannotGetPrice:
-            logger.warning("Cannot get network ether price", exc_info=True)
-            eth_price = 0
         balances_with_usd = []
-        price_token_addresses = [balance.get_price_address() for balance in balances]
-        token_eth_values_with_timestamp = (
-            self.price_service.get_token_cached_eth_values(price_token_addresses)
-        )
-        for balance, token_eth_value_with_timestamp in zip(
-            balances, token_eth_values_with_timestamp
-        ):
-            token_eth_value = token_eth_value_with_timestamp.eth_value
-            token_address = balance.token_address
-            if not token_address:  # Ether
-                fiat_conversion = eth_price
-                fiat_balance = fiat_conversion * (balance.balance / 10**18)
-            else:
-                fiat_conversion = eth_price * token_eth_value
-                balance_with_decimals = balance.balance / 10**balance.token.decimals
-                fiat_balance = fiat_conversion * balance_with_decimals
 
+        for balance in balances:
             balances_with_usd.append(
                 BalanceWithFiat(
                     balance.token_address,
                     balance.token,
                     balance.balance,
-                    token_eth_value,
-                    token_eth_value_with_timestamp.timestamp,
-                    round(fiat_balance, 4),
-                    round(fiat_conversion, 4),
+                    0.0,
+                    datetime.utcfromtimestamp(0),
+                    0.0,
+                    0.0,
                     FiatCode.USD.name,
                 )
```

Comment:
@hectorgomezv just to inform you, the transaction service will return `eth_value` as 0, `timestamp` when was calculated as 1970-01... (epoch 0), `fiat_balance` 0 and `fiat_conversion`  as 0. 
This is affecting the endpoint  `/v1/safes/{address}/balances/usd/`

Reply:
Thank you @moisses89! No problem at all from the CGW perspective, the service is not using that endpoint anymore.